app-dicts/aspell-en/aspell-en-2018.04.16.0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+ASPELL_LANG="English (US, British, Canadian)"
+ASPELL_VERSION=6
+
+inherit aspell-dict-r1 versionator
```

Comment:
Had you ran `repoman full -dx .` before committing, as you should, you'd be aware of versionator.eclass being deprecated. Use 'eapi7-ver' instead.

Reply:
I'm aware, but aspell eclass is using still EAPI 6, so changing the EAPI to 7 breaks the ebuild. Is there any other way around aside still using EAPI 6?

Reply:
Use 'eapi7-ver.eclass' instead ;)

Reply:
` * ERROR: app-dicts/aspell-en-2018.04.16.0::gentoo failed (depend phase):
 *   Unknown EAPI 7
 * 
 * Call stack:
 *                       ebuild.sh, line 635:  Called source '/usr/portage/app-dicts/aspell-en/aspell-en-2018.04.16.0.ebuild'
 *   aspell-en-2018.04.16.0.ebuild, line   9:  Called inherit 'aspell-dict-r1' 'eapi7-ver'
 *                       ebuild.sh, line 326:  Called __qa_source '/usr/portage/eclass/aspell-dict-r1.eclass'
 *                       ebuild.sh, line 111:  Called source '/usr/portage/eclass/aspell-dict-r1.eclass'
 *           aspell-dict-r1.eclass, line  39:  Called die
 * The specific snippet of code:
 *   		die "Unknown EAPI ${EAPI:-0}"
`

Reply:
It should work with EAPI=6. eapi7-ver.eclass is meant to be used when transitioning from versionator.eclass to EAPI-7, where these version modifiers are built-in.

Reply:
I asked already about this over the [forums](https://forums.gentoo.org/viewtopic-t-1089746-highlight-.html) and the conclusion was to regress to EAPI 6, since the inerited eclass wasn't supporting 7.

Reply:
I'm not sure if you understood this or not, but you use EAPI=6 and inherit eapi7-ver instead of versionator, then fix the $(replace_...) with ver_rs from eapi7-ver. That's the correct way, and it **should** work.

Reply:
Ah! My bad then, from repoman's message: 'versionator' to 'eapi7-ver (built-in since EAPI 7)', I understood I needed to use EAPI 7 in order to ve able to use eapi7-ver, I wasn't aware I could call it from a lower EAPI.